src/api/chainHead_unstable_storage.md

```diff
@@ -5,12 +5,16 @@
 - `followSubscription`: An opaque string that was returned by `chainHead_unstable_follow`.
 - `hash`: String containing an hexadecimal-encoded hash of the header of the block whose storage to fetch.
 - `key`: String containing the hexadecimal-encoded key to fetch in the storage.
-- `childKey`: `null` for main storage look-ups, or a string containing the hexadecimal-encoded key of the trie key of the trie that `key` refers to. **TODO**: I don't know enough about child tries to design this properly
+- `childTrie`: `null` for main trie look-ups, or a string containing the hexadecimal-encoded key of the key of the trie where to perform the look-up.
 - `networkConfig` (optional): Object containing the configuration of the networking part of the function. See [here](./api.md) for details. Ignored if the JSON-RPC server doesn't need to perform a network request. Sensible defaults are used if not provided.
 
 **Return value**: String containing an opaque value representing the operation.
 
-The JSON-RPC server must start obtaining the value of the entry with the given `key` (and possibly `childKey`) from the storage.
+The JSON-RPC server must start obtaining the value of the entry with the given `key` from the main trie of the storage. If `childTrie` is not `null`, then the entry must instead be loaded from the child trie whose key is `childTrie`. If the requested child trie doesn't exist, it should be treated as if it was an empty trie.
+
+> **Note**: When `childTrie` is `null`, querying a `key` that starts with the ASCII-encoded string `:child_storage:` *must* produce a `null` value. This behavior is different from the behavior of the `ext_storage_get_version_1` host function in the client-host interface, where querying a key starts with the ASCII-encoded string `:child_storage:` might return the hash of the root of a child trie.
+
+> **Note**: The `childTrie`, if not `null`, doesn't need to be prefixed with `:child_storage:` or `:child_storage:default:`. In other words, any child trie key is potentially valid, and not just ones that start with a specific prefix.
```

Comment:
```suggestion
> **Note**: The `childTrie`, if not `null`, doesn't need to be prefixed with `:child_storage:default:`. In other words, any child trie key is a valid default child trie key, and not just ones that start with a specific prefix.
```
or something similar, I mean the tradeoff with not having the prefix is that we target only `:child_storage:default:` and nothing else.